static/webview.html

```diff
@@ -30,6 +30,11 @@
         cursor: pointer;
         fill: var(--vscode-textLink-foreground);
       }
+      
+      .inactive {
+        opacity: 0.1;
+        transition: all 0.15s ease-out;
+      }
```

Comment:
Looks like ideally we will need to have not only separate JS files but also CSS ones.

We could perhaps go with deeper structure like:

```
/graphs
  /default
    index.js
    style.css
  /obsidian-like
    index.js
    style.css
```

But I think we will be fine without doing such change at this point. It can wait (or of course feel free if you prefer to do it now yourself).